src/overlays/actors/ovl_En_M_Fire1/z_en_m_fire1.c

```diff
@@ -8,39 +8,37 @@ void EnMFire1_Init(Actor* thisx, GlobalContext* globalCtx);
 void EnMFire1_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnMFire1_Update(Actor* thisx, GlobalContext* globalCtx);
 
-const ActorInit En_M_Fire1_InitVars = {
-    ACTOR_EN_M_FIRE1,
-    ACTORCAT_MISC,
-    FLAGS,
-    GAMEPLAY_KEEP,
-    sizeof(EnMFire1),
-    (ActorFunc)EnMFire1_Init,
-    (ActorFunc)EnMFire1_Destroy,
-    (ActorFunc)EnMFire1_Update,
-    NULL
-};
+const ActorInit En_M_Fire1_InitVars = { ACTOR_EN_M_FIRE1,
+                                        ACTORCAT_MISC,
+                                        FLAGS,
+                                        GAMEPLAY_KEEP,
+                                        sizeof(EnMFire1),
+                                        (ActorFunc)EnMFire1_Init,
+                                        (ActorFunc)EnMFire1_Destroy,
+                                        (ActorFunc)EnMFire1_Update,
+                                        NULL };
```

Comment:
This one was missed by your regex, because I removed the (ActorFunc) cast for the NULL. Sorry.

Reply:
No problems. The good thing is you told us about this. Thanks you